dali/pipeline/operators/reader/loader/CMakeLists.txt

```diff
@@ -23,6 +23,6 @@ set(DALI_SRCS ${DALI_SRCS} PARENT_SCOPE)
 # we don't want to test Caffe2 reader if LMDB is not present
 if (BUILD_TEST AND BUILD_LMDB)
   # get all the test srcs
-  file(GLOB tmp *_test.cc)
+  file(GLOB tmp *_test.cc file_loader.cc)
```

Comment:
This way, the `file_loader.cc` is also built in `libdali.so`. I don't know if it's on purpose, but I guess we don't need file loader in actual library

Reply:
... probably not, we need to export its symbols from DALI so I guess.